database/migrations/2019_03_13_170201_add_fields_to_devices_table3.php

```diff
@@ -14,7 +14,7 @@ public function up()
 	{
 		Schema::table('devices', function(Blueprint $table)
 		{
-      $table->string('ap_name', 255)->nullable();
```

Comment:
The changes in this file are from a different branch, right? Doesn't seem like they should be merged.

Reply:
Yeah should be removed from commit.